Makefile

```diff
@@ -12,6 +12,22 @@ test:
 	.venv/bin/py.test tests
 
 clean:
-	rm -Rf .venv 
+	rm -Rf .venv
 
-.PHONY: install test
+# Works with bash and linux
+load-tests:
+	cp conf/*.ini .
+	curl -o reingart.zip https://www.sistemasagiles.com.ar/soft/pyafipws/reingart.zip
+	python -m zipfile -e reingart.zip .
+
+sign-tra:
+	python -m pyafipws.wsaa
+
+sign-cert:
```

Comment:
This one authorizes an electronic invoice, maybe `sample-invoice:` or `try-invoice:` would be a better name 
BTW, the cert is never signed, and the access ticket is created in the wsaa module.

You should add this targets to .PHONY as they don't generate code files (beside access token and sample json invoice that are temporal data)

Reply:
Yeah, I'll add the targets to .PHONY as well